examples/step-33/step-33.cc

```diff
@@ -2277,7 +2277,6 @@ namespace Step33
 
     triangulation.execute_coarsening_and_refinement();
 
-    dof_handler.clear();
     dof_handler.distribute_dofs(fe);
```

Comment:
I missed this change. Did you test that that actually works?

Reply:
Dear Wolfgang, thank you for your careful consideration! Actually there are two same lines `dof_handler.clear();` in `step-33.cc`. I accidentally removed the wrong one. I have updated the PR now. However, I have tested and found that the program works correctly without this `clear()` call, but it is not obvious and therefore requires separate consideration.

Reply:
Thanks for the explanation. As a general rule, keep every pull request to one issue. If your PR is called "Fix some minor typos", then keep changes the code flow out of it and put it into a separate PR :-)